Extensions/ParticleSystem/JsExtension.cpp

```diff
@@ -51,6 +51,7 @@ class ParticleSystemJsExtension : public gd::PlatformExtension {
         .SetFunctionName("setAngle")
         .SetGetter("getAngle");
     conditions["ParticleSystem::EmitterAngle"].SetFunctionName("getAngle");
+    conditions["ParticleSystem::ParticleType"].SetFunctionName("getRendererType");
```

Comment:
Note that if you follow my advice of using `AddExpressionAndCondition` you will also need a line for the expression:
`strExpressions["ParticleKind"].SetFunctionName("getRendererType");`